Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -395,10 +382,12 @@ static void convertContentToRootView(const LocalFrameView& view, Vector<Selectio
                 postLayoutData.caretColor = CaretBase::computeCaretColor(editableRoot->renderer()->style(), editableRoot.get());
         }
 
-        if (RefPtr editableRootOrFormControl = enclosingTextFormControl(selection.start()) ?: selection.rootEditableElement()) {
+        if (RefPtr editableRootOrFormControl = enclosingTextFormControl(selection.start()) ?: selection.rootEditableElement())
             visualData.selectionClipRect = rootViewInteractionBounds(*editableRootOrFormControl);
-            postLayoutData.editableRootIsTransparentOrFullyClipped = result.isContentEditable && isTransparentOrFullyClipped(*editableRootOrFormControl);
-        }
+
+        if (RefPtr formControl = enclosingTextFormControl(selection.start()))
+            postLayoutData.editableRootIsTransparentOrFullyClipped = result.isContentEditable && isTransparentOrFullyClipped(*formControl);
```

Comment:
Would this also mean that we're now computing `editableRootIsTransparentOrFullyClipped` twice on iOS?